### DISCUSSION TITLE
Can't find how to add new model after UI update

### DISCUSSION THREAD

**Discussion by skobkin** [open-webui/open-webui#7314] [Q&A] [answered]
After latest UI update(s?) I can't find how to download new model from Ollama library or upload a GGUF file.

I just don't see any "add" element on the "Admin" -> "Settings" -> "Models" page anymore. Also tried checking other places for that. No luck.

I'm logged as user with "admin" role.

top part:
![image](https://github.com/user-attachments/assets/08d656b5-db29-4a5d-8a7a-23de5d0b6548)

bottom part:
![image](https://github.com/user-attachments/assets/acf0fd7d-08c6-46e2-9b01-f49bc7d830c0)

**Comment by dpk-it** [comment, marked as answer]
Discussion https://github.com/open-webui/open-webui/discussions/7054

Go to 

```
Admin Panel
    ➡️ Settings
        ➡️Connections
             ➡️ Manage Ollama API connections
                 ➡️ Manage (🔧 icon) 
```
![image](https://github.com/user-attachments/assets/1aed8eb8-bc90-4b35-af70-fc8e22711152)

**Reply by barryd15** [reply]
Thanks for this. Wish I could also heart the excellent feedback from @Nurb4000, but #7054 is now locked.

**Comment by skobkin** [comment]
In case someone is wondering like me, here's a short version of what's going on in #7054 relating to this discussion:

Go to 

```
Admin Panel
    ➡️ Settings
        ➡️Connections
             ➡️ Manage Ollama API connections
                 ➡️ Manage (🔧 icon) 
```
![image](https://github.com/user-attachments/assets/1aed8eb8-bc90-4b35-af70-fc8e22711152)

Can't say that I like specific UI decisions like icons instead of buttons with text, but reasons behind this new design makes sense.

**Comment by Qualzz** [comment]
It's a bit... confusing at least.

Having to dig through GitHub after spending seven minutes aimlessly clicking around just to figure out how to delete a model suggests that something isn't quite intuitive.

The most logical place to look for a delete option would be on the page listing the models. However, there’s an added layer of confusion between deleting a model created in OpenWeb UI and the underlying Ollama model.

This whole process seems like it needs reevaluation—it doesn’t feel like its final, polished version.

To delete a model, you have to navigate to your profile, then settings, followed by admin settings. From there, you resist the instinct to click "models" and instead go to "Connections." Then, you’re left guessing which of the two settings icons to choose:
![image](https://github.com/user-attachments/assets/54262ed2-0324-4620-82e1-93bc23e44364)

A good usability test would be asking a random relative to sit in front of the screen. After explaining the basics ("This is a model called Llama, like ChatGPT. Now, find a way to delete it."), see if they can figure it out. If even one out of ten succeeds, it’d be surprising.

**Reply by loktar00** [reply]
I just ran into this this it'ss extremely buried and confusing placement honestly. I can see why it's there to some extent, or what the developers were thinking, since it's an Ollama function not a openwebui function, but on the other hand it also feels really out of place in this area. I'd expect to see under models -> Ollama models -> add/configure.

**Reply by skobkin** [reply]
A good compromise could be something like having a hint on the "Workspace" and "Admin -> Settings -> Models" some kind of hint like: "If you're looking to manage connected back-end model list, click here (link to appropriate settings page)".

**Comment by visuttaja** [comment]
Absolutely agree about a bit strange decision to split interface location in two and crypt names. I watched Dave's Garage youtube video where it was centralized model managing was still used. It looked so Great! No it is you know like in windwos having almost sensible Control Panel and then totally destroyed Settings gui. Both at the same time confusing everyone from developers to users. I would not start shuffling with features at this phase of product life. I mean (usually) the focus should be to make core robust and easy to install before you start focusing shifting ui controls around. Not that it would not look cool overall though. This is just a not so strong opinion. I hope no one gets hurt. Here is the example. Java has audio api which thinks output is source and input is target. Developer's did not bother to think from user's point of view. It really keeps hurting the whole thing as people always first think like input was to device and output from it and it takes time to find it out, accept and get used to think like a mixer inside was thinking.
